extensions/amazon-aws/src/components/searchbar/aws-profile-dropdown.tsx

```diff
@@ -140,8 +148,19 @@ const useProfileOptions = (): ProfileOption[] => {
       configFile[name]?.region ||
       credentialsFile[name]?.region ||
       (includeProfile && configFile[includeProfile]?.region);
-
-    return { ...config, region, name };
+    const sso_start_url =
```

Comment:
As suggested above, can you test if aws-sdk will handle start-url from SSO token provider configuration gracefully? Basically, testing a shortcut creation with a SSO token provider config and not the legacy one.

Reply:
Tested and this implementation works, apart from the syntax error breaking `npm build`. You can resolve this.